gradle-example/cloudbuild.yaml

```diff
@@ -21,6 +21,6 @@ steps:
     entrypoint: gradle
     args: ["assemble"]
   - name: gcr.io/cloud-builders/docker
-    args: ["build", "-t", "gcr.io/$PROJECT_ID/hello-spring", "--build-arg=JAR_FILE=build/libs/hellospring-1.0.0.jar", "."]
-images: ["gcr.io/$PROJECT_ID/hello-spring"]
-# [END cloudbuild_gradle]
\ No newline at end of file
+    args: ["build", "-t", "my_host_location-docker.pkg.dev/$PROJECT_ID/my_repository/hello-spring", "--build-arg=JAR_FILE=build/libs/hellospring-1.0.0.jar", "."]
```

Comment:
This change is causing the test of this configuration to fail because "my_host_location" doesn't exist.

Reply:
I've updated this PR to use the _REPO_NAME substitution variable, and updated the triggers in the testing project to suit.